inc/admin.php

```diff
@@ -269,6 +269,14 @@ function settings_screen( $post_id, $strings, $args ) {
 	}
 	?>
 <div class="wrap">
+	<?php if ( file_exists( ABSPATH . '/ads.txt' ) ) { ?>
```

Comment:
This check won't work I'm afraid.

`ABSPATH` points to the directory of the WordPress install, which isn't necessarily the same as the root directory of the website. See this guide for [running wordpress in a subdirectory](https://wordpress.org/documentation/article/giving-wordpress-its-own-directory/).

To use my site as an example, this would check for `/path/to/files/wp/ads.txt` rather than `/path/to/files/ads.txt`.

I think there would be some weirdness around multisite installs too.

Reply:
Yeah, that makes sense, I'll try to fix the issue

Reply:
I added a quick comment with some pseudo code on the original issue https://github.com/10up/ads-txt/issues/48#issuecomment-1459104883

Reply:
@peterwilsoncc I'm trying to check the file following your suggestions but somehow when I try to get the file via `wp_remote_request` request I'm getting empty headers data, I've pushed the latest changes on this branch, could you take a look, please? Thanks

Reply:
I'm unable to reproduce an error with empty headers, this is what I am seeing in xdebug using the file_exists line as a break point.

<img width="1167" alt="Screen Shot 2023-04-03 at 1 49 55 pm" src="https://user-images.githubusercontent.com/519727/229407289-af49cb09-b5e6-4195-b8d0-0698d952448f.png">

Reply:
Thanks @peterwilsoncc I was able to test header data today

This is the header response from ads.txt file
<img width="699" alt="Screenshot 2023-04-26 at 8 56 17 PM" src="https://user-images.githubusercontent.com/11072036/234617189-1e993faa-c979-4682-96ce-b446afdc527f.png">

and this from adstxt post type
<img width="686" alt="Screenshot 2023-04-26 at 8 56 30 PM" src="https://user-images.githubusercontent.com/11072036/234617636-fcd28797-a757-47e8-bb66-7ef5ef391789.png">

Can we check the file with the `Content-Length` and/or `ETag` properties as these are not present in the post-type response? although I don't have much experience with server file system! please let me know your feedback. Thanks